beanstalk_worker/worker_start_test.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+import beanstalkc
+from binascii import hexlify
+import hashlib
+import json
+from subprocess import Popen
+from subprocess import PIPE
+import re
+import time
+import logging
+import sys
+import os
+import config
```

Comment:
lets group `from` and direct `import` separate, as a coding guidelines it would be good if import alphabetically